m2c/translate.py

```diff
@@ -2075,7 +2076,14 @@ def format(self, fmt: Formatter) -> str:
         ) or isinstance(dest, (ArrayAccess, LocalVar, SubroutineArg)):
             # Known destination; fine to elide some casts.
             source = elide_literal_casts(source)
-        return format_assignment(dest, source, fmt)
+        line = format_assignment(dest, source, fmt)
+        # Used in struct copy statements
```

Comment:
Drop this comment, it's mostly irrelevant it is used and it's confusing what this comment refers to exactly